src/vrl/selectielijst/constants.py

```diff
@@ -6,6 +6,7 @@
 class ArchiefNominaties(DjangoChoices):
     bewaren = ChoiceItem('bewaren', _("Bewaren"))
     vernietigen = ChoiceItem('vernietigen', _("Vernietigen"))
+    bewaren_met_uitzondering = ChoiceItem('bewaren_met_uitzondering', _("Bewaren met uitzondering van zie toelichting"))
```

Comment:
this one should be added to the constant in `zds_schema.constants` instead, right?

Reply:
zds_schema.constants already include `Archiefnominatie` constant. But it doesn't have choice 'bewaren', instead it has 'blijvend_bewaren'. Should I modify `zds_shema.constants` in order it to fit Excel data? Or should I transform data read from Excel?